lib/config/defaults.js

```diff
@@ -1053,13 +1069,15 @@ const applyOutputDefaults = (
 	);
 	F(
 		environment,
-		"forOf",
-		() => tp && optimistic(/** @type {boolean | undefined} */ (tp.forOf))
+		"optionalChaining",
+		() =>
+			tp && optimistic(/** @type {boolean | undefined} */ (tp.optionalChaining))
 	);
 	F(
 		environment,
-		"bigIntLiteral",
-		() => /** @type {boolean | undefined} */ (tp && tp.bigIntLiteral)
+		"templateLiteral",
+		() =>
+			tp && optimistic(/** @type {boolean | undefined} */ (tp.templateLiteral))
```

Comment:
Reodered like we have it in types + added `templateLiteral`, `optionalChaining` and `globalThis` support.

We don't have **all** possible ECMA features, only which used for runtime code generation, but I am fine to add more if somebody ask about it, it is easy